temba/api/v2/views.py

```diff
@@ -4441,12 +4441,12 @@ def __init__(self):
         self.applied_filters = {}
         self.request = None
 
-    def get_offset(self):
-        offset = self.request.query_params.get("offset", 0)
-        offset = int(offset) if isinstance(offset, str) and offset.isnumeric() else offset
-        if not isinstance(offset, int) or offset < 0:
-            raise Http404
-        return offset
+    def get_paginated_queryset(self, queryset):
+        pagination_class: CursorPagination = getattr(self, "pagination_class", CreatedOnCursorPagination)()
+        pagination_class.page_size_query_param = "page_size"
```

Comment:
I have a little concern about the page_size defined by the users, but let's keep that way for now.